app/helpers/application_helper.rb

```diff
@@ -9,6 +9,6 @@ def selectable(data)
   end
 
   def titleized_enum(types)
-    types.map { |x| [x[0].gsub(/_/, ' ').titleize, x[1]] }
+    types.map { |x| [x[0].gsub(/_/, ' ').titleize, x[0]] }
```

Comment:
Why the change here?
**The results:**
Before
`["Abc Dog", 1]`
After
`["Abc Dog", "abc_dog"]`

Reply:
Bugfix right?  If so, please continue.

Reply:
["Abc Dog", 1]
It is showing that 1 is not a valid mode and I tried to change x[1] to x[0] to get the "Abc Dog" and it solves the issue.

Reply:
Ok good.